website/docs/r/hdinsight_kafka_cluster.html.markdown

```diff
@@ -294,6 +318,15 @@ A `monitor` block supports the following:
 
 * `primary_key` - (Required) The Operations Management Suite (OMS) workspace key.
 
+---
+
+A `rest_proxy` block supports the following:
+
+* `security_group_name` - (Required) The AAD security group name.
+
+* `security_group_id` - (Required) The AAD security group ID.
```

Comment:
The term "AAD" is great if you're familiar with Azure, but Azure Active Directory is clearer, so it's worth using that instead here